web/package.json

```diff
@@ -35,5 +36,23 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
```

Comment:
Thanks @nik72619c for the pr, but I want to tell you that somebody already filed a pr by adding eslint and prettier ,So if you file a pr containing that code which is already there in a pr #21 , It will make that pr useless and stale. So small suggestion from my side, as I see lint stage rule depend on eslint and prettier so if you can enhance that pr #21 and add your changes to that , it will be much better for @hsluoyz to review and merge and more important thing no contribution gets wasted.

Reply:
I agree with you @shakti97, this is not the meaning of open-source to recreate another PR with extending unmerged PRs with adding some modifications. Try not to waste anyone's effort @nik72619c.

Reply:
Yup, I agree with this too @shakti97 @ani4aniket. I didn't see that there are PRs for it already. Anyways, we can close this up modify the existing PRs.